domain-specific-datasets/scripts/embed.py

```diff
@@ -0,0 +1,46 @@
+from datasets import Dataset, load_dataset
```

Comment:
Perhaps this is easier and limits the exposed logic?

https://docs.argilla.io/en/latest/practical_guides/create_update_dataset/vectors.html#add-sentence-transformers-vectors

Reply:
Yes. We should do this:

```python
from argilla.client.feedback.integrations.sentencetransformers import SentenceTransformersExtractor

dataset = ... # FeedbackDataset or RemoteFeedbackDataset

tde = SentenceTransformersExtractor(
    model="TaylorAI/bge-micro-v2",
    show_progress=True,
)

dataset = tde.update_dataset(
    dataset=dataset
    fields=None # None means using all fields
    update_records=True # Also, update the records in the dataset
    overwrite=False # Whether to overwrite existing vectors
)
```